### PR TITLE
build/configs/rtl8720e/hello/defconfig: Improve wifi throughput

### PR DESCRIPTION
**Pull request by justin-yau**
Improve wifi throughput by changing task priority and TCPIP parameters

### CODE REVIEW

File: build/configs/rtl8720e/hello/defconfig [hunk 1]
```diff
@@ -618,7 +618,7 @@ CONFIG_NET_UDP_TTL=255
 CONFIG_NET_UDPLITE=y
 CONFIG_NET_TCP=y
 CONFIG_NET_TCP_TTL=255
-CONFIG_NET_TCP_WND=14600
+CONFIG_NET_TCP_WND=26280
```
Comment (sunghan-chang) [inline build/configs/rtl8720e/hello/defconfig:621]: Could you modify this in other defconfigs like tc and loadable_apps as well?

File: build/configs/rtl8720e/hello/defconfig [hunk 2]
```diff
@@ -715,7 +715,7 @@ CONFIG_NET_MEM_SIZE=34816
 # CONFIG_NET_TCPIP_CORE_LOCKING is not set
 # CONFIG_NET_TCPIP_CORE_LOCKING_INPUT is not set
 CONFIG_NET_TCPIP_THREAD_NAME="LWIP_TCP/IP"
-CONFIG_NET_TCPIP_THREAD_PRIO=110
+CONFIG_NET_TCPIP_THREAD_PRIO=105
```
Comment (sunghan-chang) [inline build/configs/rtl8720e/hello/defconfig:718]: same as previous

File: build/configs/rtl8720e/hello/defconfig (hunk 1 above)
Comment (justin-yau) [reply build/configs/rtl8720e/hello/defconfig:621]: okok no problem

**Review by j11591 — APPROVED** [review]
LGTM

**Pull request merged**